src/test/anovos/feature_recommender/test_featrec_init.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import pandas as pd
+from anovos.feature_recommender.featrec_init import (
+    get_column_name,
+    camel_case_split,
+    recommendation_data_prep,
+)
+
+sample_data_2 = "./data/feature_recommender/test_input_fr_2.csv"
+
+
+@pytest.fixture
+def example_attr_2():
+    return pd.read_csv(sample_data_2)
```

Comment:
I think sample_data_2 can be defined inside example_attr_2() instead of a global variable.